content/md/en/docs/tutorials/get-started/build-local-blockchain.md

```diff
@@ -116,7 +122,7 @@ To start the local Substrate node:
    By default, this option also deletes all active data—such as keys, the blockchain database, and networking information when you stop the node by pressing Control-c.
    Using the `--dev` option ensures that you have a clean working state any time you stop and restart the node.
 
-1. Verify your node is up and running successfully by reviewing the output displayed in the terminal.
+2. Verify your node is up and running successfully by reviewing the output displayed in the terminal.
```

Comment:
just fyi that 1. does auto-numbering, so you shouldn't need to change step numbers.

Reply:
Yeah. There was just inconsistent use of that feature in the tutorial. The `1.` repetition was done in one place, but not others, so I just manually numbered everything when I made my other changes. The auto-numbering feature is cool, and IMHO, it'd probably be better to change all of the ordered lists to just all consistently use that feature, but I figure that can be a totally separate PR just for that.